src/extensionConflicts.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved..
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
There's no `License.txt`. How to handle this correctly?

Reply:
done

Reply:
Did you forget to push?

Reply:
yep, done.